sensor/common/virtualmachine/component_impl.go

```diff
@@ -37,6 +39,10 @@ func (c *componentImpl) Send(ctx context.Context, vm *storage.VirtualMachine) er
 		log.Warnf("Cannot send virtual machine %q to Central because Central is not reachable", vm.GetId())
 		return errox.ResourceExhausted
 	}
+	if !centralcaps.Has(centralsensor.VirtualMachinesSupported) {
+		return errox.NotImplemented.CausedBy(errCapabilityNotSupported)
+	}
```

Comment:
Let's maybe move it up - why to check if central is reachable if we know it does not support this feature?

Reply:
Capabilities are exchange in the sensor/central hello message (first two messages). It is technically possible to reach this path before we received the initial messages but I'd say very unlikely. I think depending on how the aggregator responds to `NotImplemented`, it's ok to move this message up.

Reply:
My thought was that logically we first need to confirm the connection and then the capabilities, because the latter depends on the former.

Reply:
`centralReady` is just a name for a signal that controls this component being online or offline. It may happen, that this component starts after the connection to central is established. In that case sensor would already know the capabilities of Central while it starts it's components.